### PR TITLE
Add leechcraft overlay

### PR DESCRIPTION
**Pull request by 0xd34df00d**

### CODE REVIEW

**Comment by mgorny** [comment]
The repo is missing profiles/repo_name.

**Comment by 0xd34df00d** [comment]
@mgorny somehow forgot to push that, thanks for noticing! Should be there now.

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -2639,6 +2639,18 @@
...
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>leechcraft</name>
+    <description>Overlay with LeechCraft ebuilds</description>
+    <homepage>https://github.com/0xd34df00d/leechcraft-overlay</homepage>
+    <owner type="person">
+      <email>0xd34df00d@gmail.com</email>
+      <name>Georg Rudoy</name>
+    </owner>
+    <source type="git">git://github.com/0xd34df00d/leechcraft-overlay.git</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml:2650]: Don't list unsafe protocols first.

**Review by mgorny — CHANGES_REQUESTED** [review]

File: files/overlays/repositories.xml (hunk 1 above)
Comment (0xd34df00d) [reply files/overlays/repositories.xml]: Done, moved `https://` first.

**Pull request closed without merging**